pkg/providers/aws/provider_postgres.go

```diff
@@ -203,6 +202,77 @@ func (p *AWSPostgresProvider) createRDSInstance(ctx context.Context, cr *v1alpha
 		}
 		return nil, "modify instance in progress", nil
 	}
+	// Add Tags to Aws Postgres resources
+	logrus.Infof("Adding Tags to RDS instance %s", *foundInstance.DBInstanceIdentifier)
+	// get the environment from the CR
+	defaultOrganizationTag := cr.Spec.Env.Value
+
+	// new session but don't need credentials
+	// Get the region from AvailabilityZone
+	rdsRegion := *foundInstance.AvailabilityZone
+	regionLen := len(rdsRegion)
+	rdsRegion = rdsRegion[:regionLen-1]
+	// get provider aws creds so the postgres instance can be deleted
+	rdsSvc = rds.New(session.Must(session.NewSession(&aws.Config{
```

Comment:
Why does this session not need credentials? this makes me think we can reuse the already created session.

Reply:
found it didn't work with a the created session either.